packages/main/src/config.ts

```diff
@@ -36,6 +36,7 @@ export const config = {
 	globalpingToken: process.env.GLOBALPING_TOKEN as string,
 
 	discordToken: process.env.DISCORD_TOKEN as string,
+	discordClientId: process.env.DISCORD_CLIENT_ID as string,
```

Comment:
This is actually the app ID we already use in the deploy commands so let's call it the same here, `DISCORD_APP_ID`